modules/default-requirements-central-services.adoc

```diff
@@ -82,87 +83,103 @@ Central requires Central DB to store data. The following table lists the minimum
 [id="default-requirements-central-services-scanner_{context}"]
 == Scanner
 
-Beginning with version 4.4, {product-title-short} includes two image vulnerability scanners: the StackRox Scanner and Scanner V4. The StackRox Scanner is planned to be removed in a future release, but is required for version 4.4 to perform node and platform scanning. Scanner V4 is the preferred image scanner because it provides additional features over the StackRox Scanner, such as expanded language and operating system support and data from additional vulnerability databases.
+Beginning with version 4.4, {product-title-short} includes two image vulnerability scanners: StackRox Scanner and Scanner V4. StackRox Scanner is planned to be removed in a future release, but is still required at this time to perform node and platform scanning. Scanner V4 is the preferred image scanner because it provides additional features over the StackRox Scanner, such as expanded language and operating system support and data from additional vulnerability sources.
 
 [discrete]
-== Memory and CPU requirements
+== Memory, CPU, and storage requirements
 
 [discrete]
 === StackRox Scanner
 
-The requirements in this table are based on the default of 2 replicas.
+The requirements in this table are based on the default of 3 replicas.
 
 |===
 | StackRox Scanner | CPU | Memory
 
 | *Request*
-| 2 cores
-| 3000 MiB
+| 3 cores
```

Comment:
Why are we changing the core numbers while the Scanner has been stable and no future changes will be introduced? Sorry if the information is listed somewhere else.

Reply:
Because the number of default replicas referenced above was changed. We haven't changed anything, but the documentation said the default was 2, when it's actually 3 (but please feel free to confirm this if I'm wrong)